### ISSUE TITLE
Unable to generate accessibility reports for dynamic web pages

### ISSUE THREAD

**Issue by supasani** [GoogleChrome/accessibility-developer-tools#355]
Hi,

This tools is really awesome and works perfectly fine for static web pages. But i am facing difficulty in generating reports for dynamic web pages.

What would you recommend?
Is there any update coming sometime soon which is related to the above problem?

Thanks,
Sameer

**Comment by dylanb** [comment]
@supasani what are you looking for when you say dynamic pages?

**Comment by longkubi** [comment]
![yellowcorner](https://user-images.githubusercontent.com/36709645/36498070-d7f21956-176f-11e8-8a34-e1df5a99ee0d.gif)
![folder](https://user-images.githubusercontent.com/36709645/36498075-db6d45ba-176f-11e8-8ff2-6570a8606604.gif)
![vc_loading](https://user-images.githubusercontent.com/36709645/36498462-ece04fd0-1770-11e8-9a22-2d82c5d85d17.gif)
![12](https://user-images.githubusercontent.com/36709645/36498598-54df5f04-1771-11e8-9051-25a170e53725.png)
